setup.py

```diff
@@ -88,16 +92,16 @@
         "Programming Language :: Python :: 3.8",
     ],
     description=(
-        "Delayed parallel image reading, metadata parsing, and image writing for "
-        "microscopy formats in pure Python from the Allen Institute for Cell Science."
+        "Image Reading, Metadata Conversion, and Image Writing for Microscopy Images "
+        "in Pure Python"
```

Comment:
why remove the AICS name?

Reply:
`aics` is still in the name of the library, myself and @AetherUnbound no longer work at AICS, the hope as well would be that more contributors in the future would be from open source contributors and thus I figured, "with AICS in the name of the library, we can shorten the description"